Shopify/ruby-lsp
# RuboCop Code Action "Disable for this entire file"

Follow-up to #1114, adding a code action to disable a cop for an entire file. Until that's merged, review 67fd30ef9364bca36f5f9904a7a8de03feec28c9 rather than this entire PR.

### Automated Tests

Yep, but see: https://github.com/Shopify/ruby-lsp/pull/1114/files#r1370956628

### Manual Tests

1. Check out this branch in vscode
2. Restart the LSP server
3. Edit `.rubocop.yml` to stop excluding `"test/fixtures/**/*"`
4. Edit `.vscode/settings.json` and disable `editor.formatOnSave` for `[ruby]
5. Open the `test/fixtures` directory.
6. Open files in there and run the "Disable X" code actions on the offenses.
